.gitignore

```diff
@@ -19,3 +19,5 @@ bin
 .idea/
 .vscode/
 vendor
+
+ci/jenkins/jobs/defaults.yml
```

Comment:
Could you explain why this file is updated?

Reply:
Yes, I had a discussion with @edwardbadboy and we though of adding this file to .gitignore so that we prevent the developers from pushing this file to github repo if its there in their local environment, since the change was one liner i merged it in this PR.